install.ps1

```diff
@@ -1,5 +1,9 @@
 param([switch]$WhatIf = $false)
 
+# Dot source for Get-FileEncoding
+$scriptRoot = Split-Path $MyInvocation.MyCommand.Path -Parent

```

Comment:
We have `$installDir = Split-Path $MyInvocation.MyCommand.Path -Parent` on L22. Rename this one and delete that line?